### DISCUSSION TITLE
Docstring iter_arff.py: description of filepath_or_buffer parameter - copy & paste error?

### DISCUSSION THREAD

**Discussion by anjakuchenbecker** [online-ml/river#783] [General]
Hi River Team,

during reading of your very nice API reference I came across your stream.iter_arff() function description.

You describe the filepath_or_buffer parameter as follows (docstring in _iter_arff.py_):

> """Iterates over rows from an ARFF file.
    Parameters
    ----------
    filepath_or_buffer
        Either a string indicating the location of a CSV file, or a buffer object that has a
        `read` method.
...

Maybe you mean _an ARFF file_ instead of _a CSV file_ in that context ... 

Also the filepath_or_buffer parameter in the docstring in _iter_libsvm.py_ says something about a CSV file

> """Iterates over a dataset in LIBSVM format.
    The LIBSVM format is a popular way in the machine learning community to store sparse datasets.
    Only numerical feature values are supported. The feature names will be considered as strings.
    Parameters
    ----------
    filepath_or_buffer
        Either a string indicating the location of a CSV file, or a buffer object that has a `read`
        method.
...

-> if so, you can fix that if you like :-) 
-> if not, forget my writing ;-)

Cheers (and thank you for the great river lib)
Anja

**Comment by MaxHalford** [comment]
Hey @anjakuchenbecker, thanks for the kind words! And thanks for spotting this mistake, and even better, reporting it :). I just fixed it.

**Reply by anjakuchenbecker** [reply]
Hey @MaxHalford thanks for fixing :-)
